docs/source/manual/usermanual.rst

```diff
@@ -435,3 +435,15 @@ will be aggregated into one finding, instead of one separate finding per port.
         "bit-id": "port-classification-ip",
         "config": {"aggregate_findings": "True"}
     }
+
+Also you can configure which open ports should create findings and which ports should not. This is done by settings
+common_tcp_ports, common_udp_ports, sa_tcp_ports and/or db_tcp_ports. As an example:
```

Comment:
What does `sa_tcp_ports` mean? What is the difference between the four setting lists? 

Also, from this wording it's unclear whether defining these ports here means that they will trigger findings or not. Maybe rephrase to something like: _"Any port defined in `common_tcp_ports` will no longer trigger an uncommon open port finding."_

Reply:
Makes sense! Will rephrase!